src/reducers/playlists.js

```diff
@@ -0,0 +1,27 @@
+import {storeItem} from '../localStorage';
+
+/////////////////////
+// ACTION CREATORS //
+/////////////////////
+
+export function showOnlyPlaylists(playlists) {
```

Comment:
Bare et forslag: Kanskje `setShowPlaylists`  i stedet for `showOnlyPlaylists`? Og argumentet kunne hete `showPlaylists`?